stories/setup/webpack.config.babel.js

```diff
@@ -36,22 +38,11 @@ module.exports = {
       },
     ],
   },
-  // The following plugins are needed to help handle
-  // server-side imported components such as core/logger.
-  // Is this the best way??
-  plugins: [
-    new webpack.DefinePlugin({
-      CLIENT_CONFIG: JSON.stringify(getClientConfig(config)),
-    }),
-
-    new webpack.NormalModuleReplacementPlugin(
-      /config$/,
-      'core/client/config.js',
-    ),
-  ],
+  plugins: [...getPlugins({ includeCircularDependencyPlugin: false })],
```

Comment:
Well, huh. That exposed a real bug. Please add a todo comment to re-enable this plugin (it's helpful!) after https://github.com/mozilla/addons-frontend/issues/6561 is fixed.